scripts/presubmit-tests.sh

```diff
@@ -179,7 +179,7 @@ function default_build_test_runner() {
   for pkg in ${tagged_pkgs}; do
     # `go test -c` lets us compile the tests but do not run them.
     if ! capture_output "${report}" go test -c -tags="${tags}" ${pkg} ; then
-      failed=1
+      failed=1;
```

Comment:
Not sure what's the best practice for shell, but it seems we do not need this `;`?

Reply:
deleted